file_organizer_v2/docs/CLI_DEDUPE.md

```
@@ -0,0 +1,407 @@
+#### Keep Smallest Files
+Automatically keep the smallest file in each group:
+```bash
+python -m file_organizer.cli.dedupe ~/Documents --strategy smallest
+```
+
+### Batch Mode
+
+Apply automatic strategies without per-group confirmation:
+```bash
+python -m file_organizer.cli.dedupe ~/Downloads --strategy oldest --batch
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)